valmlord/gulpify
# Feature/styles tasks

## Description
This pull request introduces new plugins to the Gulp boilerplate to enhance the build process for CSS files. It includes the addition of `gulp-concat` and `gulp-rename` plugins, as well as `shorthand` and `group-css-media-queries` plugins to the `buildCSS` task.

## Changes
### Commit 1: Add gulp-concat and gulp-rename plugins
- Added `gulp-concat` and `gulp-rename` packages to `package.json`.
- Updated `buildCSS.js` to include tasks that utilize `gulp-concat` for concatenating files and `gulp-rename` for renaming files.

### Commit 2: Add shorthand and group-css-media-queries plugins to the buildCSS task
- Added `shorthand` and `group-css-media-queries` packages to `package.json`.
- Updated the `buildCSS` task to use `shorthand` for optimizing CSS properties and `group-css-media-queries` for grouping media queries.

## How to Test
1. Execute the `gulp` command to ensure all tasks are loaded correctly.
3. Test the concatenation and renaming tasks by running the relevant Gulp tasks and verifying the output files.
4. Test the `buildCSS` task by running it and verifying that CSS properties are optimized and media queries are grouped.

## Additional Notes
- These updates aim to improve the build process by adding new features for file concatenation, renaming, and CSS optimization.
